.github/workflows/esp32-mkimage.yaml

```diff
@@ -125,3 +137,22 @@ jobs:
         name: atomvm-${{ matrix.soc }}-image
         path: ./src/platforms/esp32/build/atomvm-${{ matrix.soc }}-*.img
         if-no-files-found: error
+
+    - name: "Rename and write sha256sum"
+      if: startsWith(github.ref, 'refs/tags/')
+      shell: bash
+      working-directory: src/platforms/esp32/build
+      run: |
+        ATOMVM_IMG="AtomVM-${{ matrix.soc }}-${{ github.ref_name }}.img"
+        mv atomvm-${{ matrix.soc }}.img "${ATOMVM_IMG}"
```

Comment:
Maybe use a wild card or regex to pick out the source image file.

Reply:
Avoiding wildcard makes the pipeline fail rather than handling something unexpected.

Reply:
If this is destabilizing then we can drop it.